src/styles/PowerSelect.scss

```diff
@@ -191,6 +191,10 @@
   height: 34px;
 }
 
+.PowerSelect__TriggerLabel {
+  overflow: hidden;
+  max-width: 0px;
+}
```

Comment:
We have ellipsis in the wrong place https://github.com/selvagsz/react-power-select/blob/master/src/styles/PowerSelect.scss#L42-L44

Moving those styles to `.PowerSelect__TriggerLabel` here  https://github.com/selvagsz/react-power-select/blob/master/src/styles/PowerSelect.scss#L144-L146 should fix it

Reply:
@umanghome Also, can you help me fixing the right padding to 8px in `. PowerSelect__TriggerLabel` https://github.com/selvagsz/react-power-select/blob/master/src/styles/PowerSelect.scss#L145

With table-cell layout, we don't need the extra right padding

Reply:
Done.